src/resolvers/forgetpassword.ts

```diff
@@ -0,0 +1,62 @@
+import { LoggedUserModel } from "../models/AuthUser";
+import { sendPasswordResetEmail } from '../utils/nodemailer'; 
+import BcryptUtil from '../utils/bcrypt'; 
+import { ApolloError } from 'apollo-server-express';
+import crypto from 'crypto';
+
+export const passwordResolvers = {
+  Mutation: {
+    forgetPassword: async (_: any, { email }: { email: string }) => {
+      try {
+        const user = await LoggedUserModel.findOne({ email });
+        if (!user) {
+          console.log(`User not found for email: ${email}`);
+          throw new ApolloError('User not found');
+        }
+
+        const resetToken = crypto.randomBytes(32).toString('hex');
+
+        user.resetToken = resetToken;
+        user.resetTokenExpiration = new Date(Date.now() + 3600000); // 1 hour expiry
+        await user.save();
+
+        try {
+          await sendPasswordResetEmail(email, resetToken);
+          console.log(`Password reset email sent to ${email}`);
+          return true;
```

Comment:
Instead of returning `true`, return the message like `Reset password link is sent to your email check inbox or spam` This message will be used in the frontend